man/check-os-release.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+
+import ast
+import re
+
+def read_os_release():
+    try:
+        f = open('/etc/os-release')
+    except FileNotFoundError:
+        f = open('/usr/lib/os-release')
+
+    for line_number, line in enumerate(f):
+        if m := re.match(r'([A-Z][A-Z_0-9]+)=(.*)', line):
```

Comment:
This syntax is valid only since Python 3.8. Python 3.7 is still popular and Python 3.6 is supported till 2023. Should we use a syntax compatible with older Python releases instead?

Reply:
given it's an example rather than something that has to be run by users, imho it's fine

Reply:
maybe add a comment to the top clarifying this though, i.e.:

```
# Requires Python >= 3.8
```

or so

Reply:
Making it work with all modern Python versions is a one line change

Reply:
I used modern syntax on purpose… That example is not directed at all readers, but only at people who use python. And I think it's fair that people who use python know python syntax, and if they need to support older python versions, they can adjust the code as necessary. With python 3.10 almost out, 3.7 is stable-3, and I don't think we need to cater to it.